lib/_sidebar.erb

```diff
@@ -1,57 +1,35 @@
-    <!-- Sidebar -->
-    <div class="container">
-      <div class="col-sm-2 sidebar">
-        <nav class="nav-sidebar">
-          <ul class="nav tabs">
-          <% if !@site_map['Documents'].nil? %>
+    <aside class="sidebar">
+        <a href="" class="sidebar__close"></a>
+
+        <a href="/" class="logo">
+            <img src="https://minio.io/img/logo/wordmark_white.svg" alt="">
+        </a>
+
+        <ul class="doc-link">
```

Comment:
I think we should check for `@site_map['Documents']` not being nil here, as we might get an error if the user doesn't have anything listed for Documents in the .yml file.